src/audit-log-api/use-cases/RetryMessageUseCase.ts

```diff
@@ -23,16 +22,14 @@ export default class RetryMessageUseCase {
       return event
     }
 
-    const eventValidationResult = validateEventForRetry(event)
```

Comment:
Removed this because the function that gets the events already validates so this is surplus to requirements